packages/core/types/src/http/translations/admin/responses.ts

```diff
@@ -33,3 +33,58 @@ export interface AdminTranslationsBatchResponse {
     deleted: boolean
   }
 }
+
+/**
+ * Statistics for a specific locale.
+ */
+export interface AdminTranslationLocaleStatistics {
+  /**
+   * Expected number of translated fields.
+   */
+  expected: number
+  /**
+   * Actual number of translated fields.
+   */
+  translated: number
+  /**
+   * Number of missing translations.
+   */
+  missing: number
+}
+
+/**
+ * Statistics for an entity type.
+ */
+export interface AdminTranslationEntityStatistics
+  extends AdminTranslationLocaleStatistics {
+  /**
+   * Breakdown of statistics by locale.
+   */
+  byLocale: Record<string, AdminTranslationLocaleStatistics>
```

Comment:
suggestion: I think HTTP responses are typically snake_case regardless of their level in the object